clients/go/cmd/zbctl/testdata/deploy_resources.golden

```diff
@@ -41,6 +41,14 @@
         "resourceName": "drg-force-user.dmn",
         "tenantId": "<default>"
       }
+    },
+    {
+      "form": {
+        "formId": "form",
```

Comment:
❌ Your test is failing on this part. The `formId` at the moment is `Form_0w7r08e`